runtest262.mjs

```diff
@@ -19,7 +19,8 @@ const result = runTest262({
   test262Dir: 'test262',
   polyfillCodeFile: 'dist/script.js',
   expectedFailureFiles,
-  testGlobs: process.argv.slice(2)
+  testGlobs: process.argv.slice(2),
+  failOnUnexpectedPasses: true
```

Comment:
I don't think we need `failOnUnexpectedPasses`.... instead we can just change the test runner to always fail on unexpected passes. Anything unexpected should fail the test suite.

Reply:
Right - this will be fixed by updates to the runner in https://github.com/js-temporal/temporal-test262-runner. I'll wait for that to settle first and sync/rebase all this before merging.